client/src/components/LoginForm/LoginForm.js

```diff
@@ -32,14 +32,23 @@ class LoginForm extends Component {
     const response = await loginApiClient.login(email, password)
 
     if (response.status === 403) {
-      throw new SubmissionError({ email, _error: 'Unauthorized!' })
+      if (response.statusText === 'Wrong Email') {
+        throw new SubmissionError({ email: response.statusText,
+          _error: 'Incorrect credentials, please try again!!' })
+      } else if (response.statusText === 'Wrong Password') {
+        throw new SubmissionError({ password: response.statusText,
+          _error: 'Incorrect credentials, please try again!' })
+      } else {
+        throw new SubmissionError({ email,
+          _error: 'Incorrect credentials, please try again!' })
+      }
     } else {
       this.props.login({ email })
     }
   }
 
   render () {
-    const { handleSubmit } = this.props
+    const { handleSubmit, error } = this.props
```

Comment:
I assume `error` is coming from the `_error` in SubmissionError. Is that right?

Reply:
Yes, it is from the SubmissionError.